src/users/users.controller.ts

```diff
@@ -1,13 +1,31 @@
-import { Controller, Get } from '@nestjs/common';
+import { Controller, Get, Param } from '@nestjs/common';
 import { UsersService } from './users.service';
 import { User } from '@prisma/client';
+import {
+  ApiNotFoundResponse,
+  ApiOkResponse,
+  ApiOperation,
+  ApiTags,
+} from '@nestjs/swagger';
+import { UsersEntity } from './entities/users.entity';
 
+@ApiTags('users')
 @Controller('users')
 export class UsersController {
   constructor(private usersService: UsersService) {}
 
   @Get()
+  @ApiOperation({ summary: 'Get users list' })
+  @ApiOkResponse({ type: UsersEntity, isArray: true })
   getUsers(): Promise<User[]> {
```

Comment:
For this scenario check comment below: https://github.com/Ciszak-Krzysztof/leftovers-be/pull/3/files#r1665694655

For array there are two approaches:
- one would be to simply return Promise<Array<GetUserResponse>> - with this approach we are somewhat tied to one type of response on 2 endpoints
- the second would be to create a `GetUserListResponse` class and here instead of directly returning an array we return something like this:

```json
{
  "users": [ {...}, {...}, ... ]
}
```